api/api.json

```diff
@@ -147,6 +145,35 @@
         }
       ]
     },
+    "segmentUpdateForm": {
+      "description": "Form to modify an existing segment",
+      "fields": [
+        { "name": "name",
```

Comment:
not everything 😛 https://github.com/trifectalabs/roadquality/pull/13/files/3349ae90fa3131b17dacae5214d73a20fba0cec2#diff-4b95db13161ad2c8cfa9f6d16d3858a6R260